packages/chakra-ui/package-lock.json

```diff
@@ -1,8 +1,11693 @@
 {
 	"name": "@rjsf/chakra-ui",
 	"version": "2.5.1",
-	"lockfileVersion": 1,
+	"lockfileVersion": 2,
```

Comment:
I've noticed that you changed the lockfileVersion to 2. This is probably because you're using npm 7. Can you ensure you're using node v14 and npm 6 whenever you're updating the package-lock.json files? This way, we'll ensure that the whole codebase uses lockfileVersion 1 for consistency.

We probably want to eventually move things over to node 15 and npm 7 (with lockfileVersion 2), but that's probably for another PR 😄

Reply:
roger that.

Reply:
Hey @agustin107 @epicfaace , anytime soon that this PR would be considered in the release?